src/client/pages/main/components/Pics/styles.pcss

```diff
@@ -0,0 +1,54 @@
+@import 'variables.pcss';
+
+.pic {
+  height: auto;
+  width: 49%;
+  margin-bottom: var(--distance-500);
+  background-size: cover;
+  background-position: center center;
+  background-repeat: no-repeat;
+
+  img {
+    width: 100%;
+  }
+}
+
+.pics {
+  height: auto;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  padding: var(--distance-1600) 0;
+}
+
+.textWrap {
+  position: relative;
+  height: 0;
+  width: 100%;
+  top: calc(-50% - 2em);
+  text-align: center;
+}
+
+.text {
+  font-weight: var(--text-weight-700);
+  color: white;
+  font-size: 1.5em;
+  line-height: 3em;
```

Comment:
You don't change font-size or line-height dynamically, therefore I don't see the need to use `em`. Please use `px` in the application.